examples/ndt2/train.py

```
@@ -0,0 +1,536 @@
+            filtered_state_dict = {
+                k.replace("model.", ""): v
+                for k, v in state_dict.items()
+                if not any(k.startswith(excl) for excl in exclude_prefixes)
+            }
+
+            # Load remaining weights (encoder, embeddings, etc.)
+            missing, unexpected = model.load_state_dict(
+                filtered_state_dict, strict=False
+            )
+
+            log.info(f"Loaded pretrained model (excluding decoder).")
```

Ruff (0.15.1)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)